scripts/Actions/Explore.ts

```diff
@@ -42,10 +42,21 @@ export function hideAddedAnime() {
 }
 
 // Hides anime that are not in your list.
-export function calendarShowAddedAnimeOnly() {
+export function calendarShowAddedAnimeOnly(arn: AnimeNotifier, element: HTMLInputElement) {
 	for(let anime of findAll("calendar-entry")) {
 		if(anime.dataset.added === "false") {
 			anime.classList.toggle("hidden")
 		}
 	}
+
+	const showUserList = !Array.from(document.getElementsByClassName("calendar-entry"))
+		.some(value => value.classList.contains("hidden"));
+
+	let obj = {
+		"CalendarSettings.ShowUserList": showUserList
```

Comment:
I needed a full 5 minutes to understand what `ShowUserList` means.
I thought this was about showing a list of users (friends watching it?) in the calendar view.
It also doesn't include the word "only", which is weird because they are shown by default.
This name should be changed to something like `ShowAddedAnimeOnly`.